### ISSUE TITLE
Accessibility: MAS05A: Keyboard: Keyboard focus is not properly visible for the social links

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#37]
#A11yMAS
#MAS36
#Wipro
#vscode-website
#FireFox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Repro Steps:**
1.Open https://code.visualstudio.com/docs in edge.
2.Navigate to RHS pane contents  under "Getting Started" below the header of the page .
3.Observe keyboard focus [Bounded rectangular box] for the social links.

**Actual Result:**
Keyboard focus [Bounded rectangular box] is not visible  for the  social links "Subscribe", "Ask questions", "Follow @code", "Request features", "Report issues". [refer screenshot]

**Expected Result:**
Keyboard focus [Bounded rectangular box] is should be visible  for the social links "Subscribe", "Ask questions", "Follow @code", "Request features", "Report issues".

**User Impact:**
Users with visual and/or motor impairments who rely on keyboard may not be able to identify that the social links are focused or not.

**Screenshot:**
![no bounded rectangular box for links](https://cloud.githubusercontent.com/assets/24825441/21710353/0844e936-d40e-11e6-8fcf-d8ff6bf24854.png)

**Comment by chrisdias** [comment]
I can't reproduce this, I see the bounding rectangle on all of the social links as I tab through them.
